app/cmd/root.go

```diff
@@ -93,6 +93,21 @@ func init() {
 	RootCmd.PersistentFlags().IntVar(&config.API.GrpcServerPort, "grpc-port",
 		config.API.GrpcServerPort, "GRPC api server port")
 
+	/**========================Hare Flags ========================== **/
+
+	// N determines the size of the hare committee
+	RootCmd.PersistentFlags().IntVar(&config.HARE.N, "hare-committee-size",
+		config.API.JSONServerPort, "Size of Hare committee")
+	// F determines the max number of adversaries in the Hare committee
+	RootCmd.PersistentFlags().IntVar(&config.HARE.F, "hare-max-adversaries",
+		config.API.JSONServerPort, "Max number of adversaries in the Hare committee")
+	// SetSize determines the size of the initial set
+	RootCmd.PersistentFlags().IntVar(&config.HARE.SetSize, "set-size",
+		config.API.JSONServerPort, "Size of the initial set")
+	// RoundDuration determines the duration of a round in the Hare protocol
+	RootCmd.PersistentFlags().DurationVar(&config.HARE.RoundDuration, "round-duration-ms",
+		config.P2P.ResponseTimeout, "Duration of round in the Hare protocol")
```

Comment:
same comment as about. 
also why did you used `hare` in some places and in some you didn't ? consider be more consistent

Reply:
Fixed